xchange-bitfinex/src/main/java/org/knowm/xchange/bitfinex/BitfinexExchange.java

```diff
@@ -70,7 +70,7 @@ public void remoteInit() throws IOException, ExchangeException {
       Map<CurrencyPair, BigDecimal> lastPrices =
           Arrays.stream(dataService.getBitfinexTickers(null))
               .map(BitfinexAdapters::adaptTicker)
-              .collect(Collectors.toMap(t -> t.getCurrencyPair(), t -> t.getLast()));
+              .collect(Collectors.toMap(t -> t.getCurrencyPair(), t -> t.getLast() == null ? BigDecimal.ZERO : t.getLast()));
```

Comment:
null represents a lack of value
zero i zero, the fact that we did not find the last price does not mean it was zero

Reply:
How would you prefer to handle null value then? as this throws exception when last is null

Reply:
I would prefer to add this before the collect:

```
.fitler(t -> t.getLast() != null)
```